jenkins/version-def.sh

```diff
@@ -29,9 +29,15 @@ IFS=$PRE_IFS
 CUDF_VER=${CUDF_VER:-"0.18-SNAPSHOT"}
 CUDA_CLASSIFIER=${CUDA_CLASSIFIER:-"cuda10-1"}
 PROJECT_VER=${PROJECT_VER:-"0.4.0-SNAPSHOT"}
+RROJECT_TEST_VER=${PROJECT_TEST_VER:-"0.4.0-SNAPSHOT"}
```

Comment:
Just noticed this typo:
```suggestion
PROJECT_TEST_VER=${PROJECT_TEST_VER:-"0.4.0-SNAPSHOT"}
```

Reply:
@jlowe  Thanks a lot!